production/postgresdb.tf

```diff
@@ -6,7 +6,7 @@ module "postgres_db_production" {
   environment_name = "production"
   vpc_id =  "vpc-0ce853ddb64e8fb3c"
   db_engine = "postgres"
-  db_engine_version = "12.17"
+  db_engine_version = "16.3"
```

Comment:
No parameter group is specified on this database, so it will probably default to default.postgres-16, which forces SSL.

Reply:
This was set using the postgres16 parameter group.  However, changes are still pending maintenance window so won't be able to confirm until upgrade is complete.

Reply:
We can just add this missing configuration anyway - it won't affect the pending status.

Reply:
Adding it in now.